src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,10 +25,8 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(
-            ~TargetFrameworkMonikers.Netcoreapp, 
-            "NetFramework: NetEventSource is only part of .NET Core;" + 
-            "UapAot: RemoteExecutorConsoleApp.exe crashes trying to load netstandard.dll - StrongName issue")]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/19909", TargetFrameworkMonikers.Uap)]
```

Comment:
@CIPop I personally view it similar to formatting changes - nice to have, but not necessary. If you want to drive, please do so.
Small footnote in standards/best-practices might be valuable, but I don't have high hopes people will notice it, unless they are first-time contributors. I don't think it is worth enforcing it in any way beyond pointing it out in code reviews, similar to code formatting rules. If it sticks and is widely adopted, then great. If it doesn't ... we will have inconsistencies, unless someone is motivated to 'fix it up' from time to time ...